config/routes.rb

```diff
@@ -95,7 +95,11 @@
       post :update_image, on: :member
       get :managed, on: :collection
       get :accessible, on: :collection
+
+      resource :logo, controller: "enterprise_logo", only: [:destroy]
+      resource :promo_image, controller: "enterprise_promo_image", only: [:destroy]
```

Comment:
❤️ restful routes! why don't you follow the Rails convention regarding the controller name though?

Reply:
@sauloperez With that, do you mean that this should use the plural form? I'll make that change.

Or did you also mean that the controller name should be "logo" ("logos" pending change :point_up:) not "enterprise_logo"?

Reply:
I would simply remove the `controller` argument and 💯 with whatever might be the Rails convention. This makes things simpler in the long run.

Reply:
I renamed the controllers to `LogosController` and `PromoImagesController`, but kept the subclass `EnterpriseAttachmentController`.

Before, I was concerned that `Logos` and `PromoImages` would be vague, but maybe not. If there would be a naming clash in the future, we can rename them then.